disnake/webhook/async_.py

```diff
@@ -1188,7 +1198,7 @@ def from_url(
 
```

Comment:
Same again: should we do an isinstance check on `session` somewhere here for consistency with `SyncWebhook`?